### PR TITLE
zr-asmft304-checkout_revamp

### PR DESCRIPTION
**Pull request by roppel5ls**
I am currently re-working our checkout scripts in order to be fully ready for our site-functionality tests.

For the first part of this I focused on slowing checkout down and making it more stable - I removed pauses from the individual scripts and placed a pause between each step in the checkout mods file. I also changed some of the selectors to xpath's that are a bit more stable.

I will have more PR's to come, I just wanted to get this stable enough to run consistently.

### CODE REVIEW

File: demo/checkout/checkout_corp_canadian.js [hunk 1]
```diff
@@ -16,5 +16,5 @@ what('I can fill out a credit card application on the Canadian corporate site AS
   o.caCorpLsComboContInfo()
   o.ccPaymentInfo()
   o.caCorpComboReview()
-})
+  })
```
Comment (jeremycrussell) [inline demo/checkout/checkout_corp_canadian.js]: spacing??   Not sure if this is following a pre-determined style or not, so you can probably ignore this.

**Comment by robertvazquezlegalshield** [comment]
LGTM

File: demo/checkout/checkout_corp_canadian.js (hunk 1 above)
Comment (TrentonWalkup) [reply demo/checkout/checkout_corp_canadian.js]: I keep trying to find time to go with a linter

**Comment by roppel5ls** [comment]
Approved - @jeremycrussell , @robertvazquezlegalshield , @TrentonWalkup

**Pull request merged**